platform/nebulas/api_test.go

```diff
@@ -66,6 +68,8 @@ func TestNormalize(t *testing.T) {
 	}
 
 	if !bytes.Equal(resJSON, dstJSON) {
+		println(string(resJSON))
+		println(string(dstJSON))
```

Comment:
ca we use `"github.com/stretchr/testify/assert"` to assert easier?

Reply:
Yes, I did this PR to use the auto-gen coin file, but some tests are broke because there are missing parameters, I just fix simple things.